libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/QnAMaker/QnAMakerComponentRegistration.cs

```diff
@@ -4,30 +4,27 @@
 using System.Collections.Generic;
 using Microsoft.Bot.Builder.Dialogs.Adaptive.QnA;
 using Microsoft.Bot.Builder.Dialogs.Adaptive.QnA.Recognizers;
-using Microsoft.Bot.Builder.Dialogs.Debugging;
 using Microsoft.Bot.Builder.Dialogs.Declarative;
-using Microsoft.Bot.Builder.Dialogs.Declarative.Resolvers;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
 using Microsoft.Bot.Expressions.Properties.Converters;
 using Newtonsoft.Json;
 
 namespace Microsoft.Bot.Builder.AI.QnA
 {
-    public class QnAMakerComponentRegistration : ComponentRegistration
+    public class QnAMakerComponentRegistration : ComponentRegistration, IComponentDeclarativeTypes
     {
```

Comment:
Documentation--maybe we need to turn on lack of documentation errors again and do a general check.